src/mm-route.js

```diff
@@ -120,25 +108,17 @@ angular.module('mmRoute', [
     // Arguments:
     //   name      {String}    Name of the route; nested names separated by .
     //   [data]    {Object}    A map of route paramater names to data
-    //   [role]    {String}    The role to use in the case of multiple routes
-    //
-    function get(name, data, role) {
-      /*jshint devel: true */
-
-      // The `data` and `role` arguments are both optional. This handles the
-      // case where `data` is undefined but `role` is not
-      if (arguments.length === 2 && typeof data === 'string') {
-        role = data;
-        data = undefined;
-      }
+
+    function get( name, data ) {
+    /*jshint devel: true */
```

Comment:
In fact, there are now numerous problems with the function comments. Can you please go through and ensure each function is correctly documented.